mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1092,11 +1092,14 @@ def evaluate(
         if evaluator_config.get("_disable_candidate_model", False):
             evaluation_result = EvaluationResult(metrics=dict(), artifacts=dict())
         else:
+            if baseline_model:
+                _logger.info("Evaluating candidate model:")
```

Comment:
When I was looking at the model evaluation logs, I didn't know whether they belonged to the candidate or the baseline model. I'm adding these logs so that it would be easier for users to make that discernment.